index.js

```diff
@@ -1,47 +1,43 @@
 #!/usr/bin/env node
-
-const glob = require('glob')
-const fs = require('fs')
-const path = require('path')
-const mkdirp = require('mkdirp')
-const prettier = require('prettier')
-
-// options is optional
-const getFiles = (currentPath) => {
-    const files = glob.sync(currentPath + "/**/*.ws")
-    return files.filter(file => file.indexOf('node_modules/') < 0)
-}
-
-const compile = (content) => {
-    const keywords = JSON.parse(fs.readFileSync(path.join(__dirname, 'keywords.json'), 'utf8'))
-    for (const key in keywords) {
-        content = content.replace(new RegExp(key, "g"), keywords[key]);
-    }
-    return content
-}
-
-const run = () => {
-    console.log('HAWIDERE')
-    const currentPath = process.cwd()
-    const files = getFiles(currentPath)
-
-    console.log('DO HOBN MA ' + files.length + ' GSCHICHTLN')
-
-    files.forEach(file => {
-        file = path.normalize(file)
-        const content = fs.readFileSync(path.normalize(file), 'utf8')
-        const dir = path.join(currentPath, 'dist')
-        if (!fs.existsSync(dir)) fs.mkdirSync(dir)
-
-        const dirs = file.substring(0, file.lastIndexOf(path.sep)).replace(currentPath, '')
-        mkdirp.sync(path.join(currentPath, 'dist', dirs))
-
-        const filePath = path.join(currentPath, 'dist', file.replace(currentPath, '').replace('.ws', '.js'))
-        console.log('DRAH DI DEPATTA', file.replace(currentPath, '').replace(path.sep, '') + ' ==> ' + filePath.replace(currentPath, '').replace(path.sep, ''))
-        fs.writeFileSync(filePath, prettier.format(compile(content), { parser: 'babel' }), (error) => console.error(error))
-    })
-
-    console.log('PFIATI')
-}
-
-run()
\ No newline at end of file
+const glob = require('glob'),
+  fs = require('fs'),
+  path = require('path'),
+  prettier = require('prettier'),
+  keywords = JSON.parse(fs.readFileSync('./keywords.json'));
+
+const getFiles = () => glob.sync(process.cwd() + '/**/*.ws', { ignore: 'node_modules/'});
```

Comment:
just tested it and `node_modules/` is not being ignored. had the same issue, that's why I did this:
```
files.filter(file => file.indexOf('node_modules/') < 0)
```

Reply:
for me it is ignored... weird, i will investigate on this

Reply:
![image](https://user-images.githubusercontent.com/17516174/87637259-3e95e000-c742-11ea-8acc-1944760b42ac.png)
this is what `console.log(getFiles())` returns for me
using glob@7.1.6

Reply:
I'm using the same glob version. I put a `test.ws` file into the `node_modules/` folder and it got transpiled to `dist`. can you try that on your end?

Reply:
Oh yeah! Had to specify the exclusion path a bit different

Reply:
This now works as expected!